issue_4/contracts/GreenVault.sol

```diff
@@ -1,8 +1,19 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.8.4;
 
+import "@yield-protocol/utils-v2/contracts/access/Ownable.sol";
 import "@yield-protocol/utils-v2/contracts/token/ERC20.sol";
 
+contract FPMath {
```

Comment:
I would make this into an internal library, as it makes later for cleaner syntax. See [here](https://github.com/yieldprotocol/vault-v2/blob/a6b170f648ca9e25c9eb836bcf1de407e3f3bdf8/contracts/FYToken.sol#L20).

Reply:
ah, the library has to be *internal* for this to work. I actually started FPMath as a library, but got invalid bytecode generation